src/linalg/mod.rs

```diff
@@ -76,6 +76,11 @@ pub trait BaseVector<T: RealNumber>: Clone + Debug {
     /// Get number of elevemnt in the vector
     fn len(&self) -> usize;
 
+    /// Returns true if the vector is empty.
+    fn is_empty(&self) -> bool {
```

Comment:
I just implemented this to remove the warning, but not sure if this actually makes sense and has a use case for this library. We can just ignore that warning if this is not needed